plugins/tours/tests/phpunit/unit/setPastToursByOrderNumber.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Tests for set_past_tours_by_order_number().
+ *
+ * @package     spiralWebDb\CornerstoneTours\Tests\Unit
+ * @since       1.0.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Unit;
+
+use Mockery as m;
+use Brain\Monkey;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+use function spiralWebDb\CornerstoneTours\set_past_tours_by_order_number;
+
+/**
+ * Class Tests_SetPastToursByOrderNumber
+ *
+ * @package spiralWebDb\CornerstoneTours\Tests\Unit
+ * @group   tours
+ */
+class Tests_SetPastToursByOrderNumber extends Test_Case {
+
+	/**
+	 * Instance of WP_Query for each test.
+	 *
+	 * @var Mockery
+	 */
+	protected $query;
+
+	/**
+	 * Query vars, after parsing
+	 *
+	 * @var array
+	 */
+	public $query_vars = array();
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once TOURS_ROOT_DIR . '/src/plugin.php';
+
+		$query = m::mock( 'WP_Query' );
```

Comment:
`$query` is a local scoped variable and not a property on the object. Huh? It's only available to the `setUp()` method and not the test methods. Why? To assign it to the object, you need to `$this`:

```php
$this->query = m::mock( 'WP_Query' );
```